_docs/_latest/configure/gsuite-group-collection.md

```diff
@@ -10,33 +10,20 @@ processing by Forseti Inventory.
 
 ## Enable Domain-Wide Delegation in G Suite
 
-To enable collection of G Suite Google Groups, follow the steps below to create a
-service account just for this functionality. Read more about 
+To enable collection of G Suite Google Groups, follow the steps below to enable your existing forseti
+service account for this functionality. Read more about 
 [domain-wide delegation](https://developers.google.com/identity/protocols/OAuth2ServiceAccount?hl=en_US#delegatingauthority).
 
-### Create a service account
+### Enable DwD on a service account
 
-1. Go to
-   [Cloud Platform Console Service Accounts](https://console.cloud.google.com/iam-admin/serviceaccounts){:target="_blank"}
-   and click **Create service account**.
+1. Navigate to the [**service account**](https://pantheon.corp.google.com/projectselector/iam-admin/serviceaccounts) page
```

Comment:
No need to build this, and please also keep the target tag.

Reply:
Updated.